tests/e2e/e2e_test.go

```diff
@@ -445,23 +444,30 @@ func (s *IntegrationTestSuite) TestSuperfluidVoting() {
 	)
 }
 
-// Copy a file from A to B with io.Copy
-func copyFile(a, b string) error {
-	source, err := os.Open(a)
-	if err != nil {
-		return err
-	}
-	defer source.Close()
-	destination, err := os.Create(b)
-	if err != nil {
-		return err
-	}
-	defer destination.Close()
-	_, err = io.Copy(destination, source)
-	if err != nil {
-		return err
+func (s *IntegrationTestSuite) TestRateLimitingParam() {
+	if s.skipUpgrade {
+		s.T().Skip("Skipping IBC tests")
 	}
-	return nil
+
+	// After v15, rate limiting gets set on genesis.
+	chainA := s.configurer.GetChainConfig(0)
+	chainB := s.configurer.GetChainConfig(1)
+
+	nodeA, err := chainA.GetDefaultNode()
+	s.Require().NoError(err)
+	nodeB, err := chainB.GetDefaultNode()
+	s.Require().NoError(err)
+
+	// Need to json unparshal the params because they are stored including quotes
+	paramA := nodeA.QueryParams(ibcratelimittypes.ModuleName, string(ibcratelimittypes.KeyContractAddress))
+	json.Unmarshal([]byte(paramA), &paramA)
+	paramB := nodeB.QueryParams(ibcratelimittypes.ModuleName, string(ibcratelimittypes.KeyContractAddress))
+	json.Unmarshal([]byte(paramB), &paramB)
+
+	// When upgrading to v15, we want to make sure that the rate limits have been set.
+	quotas, err := nodeA.QueryWasmSmartArray(paramA, `{"get_quotas": {"channel_id": "any", "denom": "ibc/E6931F78057F7CC5DA0FD6CEF82FF39373A6E0452BF1FD76910B93292CF356C1"}}`)
+	s.Require().Len(quotas, 2)
+	s.Require().NoError(err)
```

Comment:
Should we repeat this check for both chains A and B?

The reason is that for chain A we overwrite the quotas in `TestIBCTokenTransferRateLimiting` so making this check against `nodeA` only does not validate the correctness of the upgrade